connector/keystone/keystone.go

```diff
@@ -111,11 +113,12 @@ var (
 // Open returns an authentication strategy using Keystone.
 func (c *Config) Open(id string, logger log.Logger) (connector.Connector, error) {
 	return &conn{
-		c.Domain,
-		c.Host,
-		c.AdminUsername,
-		c.AdminPassword,
-		logger,
+		Domain:        c.Domain,
+		Host:          c.Host,
+		AdminUsername: c.AdminUsername,
+		AdminPassword: c.AdminPassword,
+		Logger:        logger,
+		client:        http.DefaultClient,
```

Comment:
The test's problem is that they didn't use the `Open` method and manually create connector instances. You should add `http.DefaultClient` to every `conn{}` in tests.